src/Compilers/CSharp/Test/Semantic/Semantics/DelegateTypeTests.cs

```diff
@@ -13109,5 +13161,512 @@ public static void Main()
 """;
             CompileAndVerify(source, expectedOutput: "3").VerifyDiagnostics();
         }
+
+        [Theory]
+        [InlineData("sbyte")]
+        [InlineData("byte")]
+        [InlineData("short")]
+        [InlineData("ushort")]
+        [InlineData("int")]
+        [InlineData("uint")]
+        [InlineData("long")]
+        [InlineData("ulong")]
+        [InlineData("nint")]
+        [InlineData("nuint")]
+        [InlineData("float")]
+        [InlineData("double")]
+        [InlineData("decimal")]
+        [InlineData("E", "E.FIELD", "FIELD")]
+        [InlineData("bool", "true", "True")]
+        [InlineData("char", "'a'", "a")]
+        [InlineData("string", @"""a string""", "a string")]
+        [InlineData("C", "null", "")]
+        public void LambdaDefaultParameter_AllConstantValueTypes(string parameterType, string defaultValue = "0", string expectedOutput = "0")
+        {
+            var source = $@"
```

Comment:
Consider using `$$"""` literals.

Reply:
Good idea. Used them across all new tests added in this PR: 5ff45b7071f030e6c2125bd865e60d68a4112c4d